src/sentry/static/sentry/app/components/charts/series/mapSeries.tsx

```diff
@@ -0,0 +1,18 @@
+import 'echarts/lib/chart/map';
+import 'echarts/map/js/world';
+import {EChartOption} from 'echarts';
+
+export default function MapSeries(
+  props: EChartOption.SeriesMap = {}
+): EChartOption.SeriesMap {
+  return {
+    roam: true,
+    itemStyle: {
+      // TODO(ts): label doesn't seem to exist on the emphasis? I have not
+      //           verified if removing this has an affect on the world chart.
+      emphasis: {label: {show: false}} as any,
```

Comment:
Looks like emphasis is an option on the [map-series](https://echarts.apache.org/en/option.html#series-map.emphasis) directly and not inside of itemStyle.

Reply:
Going to leave it as is for now and come back to this later. I could see this going away since dashboards might go away (?).